pkg/discovery/monitoring/kubelet/kubelet_monitor.go

```diff
@@ -162,44 +162,62 @@ func (m *KubeletMonitor) generateThrottlingMetrics(metricFamilies map[string]*dt
-// "container_cpu_cfs_throttled_periods_total" and "container_cpu_cfs_periods_total".
-// It deciphers the container id from the labels on the metric and merges the two for
+// parseMetricFamilies parses the incoming prometheus format metric from four metric families
+// "container_cpu_cfs_throttled_periods_total", "container_cpu_cfs_periods_total", "container_spec_cpu_quota"
+// and "container_spec_cpu_period".
+// It deciphers the container id from the labels on the metric and merges the four for
 // each container into "type throttlingMetric struct".
 // Example:
 // in:
+// # HELP container_cpu_cfs_periods_total Number of elapsed enforcement period intervals.
 // # TYPE container_cpu_cfs_periods_total counter
 // container_cpu_cfs_periods_total{container="metallb-speaker",id="/kubepods/pod8266a379-dd56-42f8-8af0-19fc0d8ea3af/8e1a2ff0f116c9d086af53cbd7430dced0e70fed104aea79e3891870564aed38",image="sha256:8c49f7de2c13b87026d7afb04f35494e5d9ce6b5eeeb7f8983d38e601d0ac910",name="k8s_metallb-speaker_metallb-speaker-m29mf_ccp_8266a379-dd56-42f8-8af0-19fc0d8ea3af_16",namespace="ccp",pod="metallb-speaker-m29mf"} 10 1616975907597
+// # HELP container_cpu_cfs_throttled_periods_total Number of throttled period intervals.
 // # TYPE container_cpu_cfs_throttled_periods_total counter
 // container_cpu_cfs_throttled_periods_total{container="metallb-speaker",id="/kubepods/pod8266a379-dd56-42f8-8af0-19fc0d8ea3af/8e1a2ff0f116c9d086af53cbd7430dced0e70fed104aea79e3891870564aed38",image="sha256:8c49f7de2c13b87026d7afb04f35494e5d9ce6b5eeeb7f8983d38e601d0ac910",name="k8s_metallb-speaker_metallb-speaker-m29mf_ccp_8266a379-dd56-42f8-8af0-19fc0d8ea3af_16",namespace="ccp",pod="metallb-speaker-m29mf"} 5 1616976197080
+// # HELP container_spec_cpu_quota CPU quota of the container.
+// # TYPE container_spec_cpu_quota gauge
+// container_spec_cpu_quota{container="metallb-speaker",id="/kubepods/pod8266a379-dd56-42f8-8af0-19fc0d8ea3af/8e1a2ff0f116c9d086af53cbd7430dced0e70fed104aea79e3891870564aed38",image="sha256:8c49f7de2c13b87026d7afb04f35494e5d9ce6b5eeeb7f8983d38e601d0ac910",name="k8s_metallb-speaker_metallb-speaker-m29mf_ccp_8266a379-dd56-42f8-8af0-19fc0d8ea3af_16",namespace="ccp",pod="metallb-speaker-m29mf"} 10000 1629775344665
+// # HELP container_spec_cpu_period CPU period of the container.
+// # TYPE container_spec_cpu_period gauge
+// container_spec_cpu_period{container="metallb-speaker",id="/kubepods/pod8266a379-dd56-42f8-8af0-19fc0d8ea3af/8e1a2ff0f116c9d086af53cbd7430dced0e70fed104aea79e3891870564aed38",image="sha256:8c49f7de2c13b87026d7afb04f35494e5d9ce6b5eeeb7f8983d38e601d0ac910",name="k8s_metallb-speaker_metallb-speaker-m29mf_ccp_8266a379-dd56-42f8-8af0-19fc0d8ea3af_16",namespace="ccp",pod="metallb-speaker-m29mf"} 100000 1629775404902
 //
 //out:
 // map[string]*throttlingMetric{
 //		"ccp/metallb-speaker-m29mf/metallb-speaker": {
 //			cpuThrottled: 5,
 //			cpuTotal:     10,
+//          cpuQuota:     10000,
+//          cpuPeriod:    100000,
```

Comment:
Nit: indent not aligned with lines 206-207?

Reply:
The indentations are correct in the IDE:
![Screen Shot 2021-08-25 at 10 53 58 AM](https://user-images.githubusercontent.com/23689754/130816745-6af693b1-0324-491a-85c7-291b8dc7e544.png)

Looks it's not properly displayed in github PR here